includes/class-wc-stripe-api.php

```diff
@@ -173,21 +190,42 @@ public static function get_idempotency_key( $api, $method, $request ) {
 	 * @throws WC_Stripe_Exception
 	 */
 	public static function request( $request, $api = 'charges', $method = 'POST', $with_headers = false ) {
-		WC_Stripe_Logger::log( "{$api} request: " . print_r( $request, true ) );
-
 		$headers = self::get_headers();
 
 		$idempotency_key = apply_filters( 'wc_stripe_idempotency_key', self::get_idempotency_key( $api, $method, $request ), $request );
 		if ( $idempotency_key ) {
 			$headers['Idempotency-Key'] = $idempotency_key;
 		}
 
+		$request = apply_filters_deprecated(
+			'woocommerce_stripe_request',
+			[ $request, $api ],
+			'9.6.0',
+			'wc_stripe_request',
+			'The woocommerce_stripe_request filter is deprecated since WooCommerce Stripe Gateway 9.6.0, and will be removed in a future version. Use wc_stripe_request instead.'
+		);
+
+		/**
+		 * Filters the request body sent to the Stripe API.
+		 *
+		 * @since 9.6.0
+		 *
+		 * @param array $request The request body from the deprecated filter.
+		 * @param string $api The Stripe API endpoint.
+		 *
+		 * @return array The request body to send to Stripe API.
+		 */
+		$request = apply_filters( 'wc_stripe_request', $request, $api );
```

Comment:
Related to my previous comment, I think the new filter should be `wc_stripe_request_body` to comply with the original version and to match it to the body param context.